pyproject.toml

```diff
@@ -39,28 +39,37 @@ scipy = "~1.14.0"
 #----------------------------------------
 #geoh5py = {version = "~0.10.0-alpha.1", source = "pypi", allow-prereleases = true}
 geoh5py = {git = "https://github.com/MiraGeoscience/geoh5py.git", rev = "release/0.10.0"}
+#geoh5py = {version = "~0.10.0-alpha.1", allow-prereleases = true}
 
 #octree-creation-app = {version = "~0.2.0-alpha.1", source = "pypi", allow-prereleases = true}
 octree-creation-app = {git = "https://github.com/MiraGeoscience/octree-creation-app.git", rev = "release/0.2.0"}
+#octree-creation-app = {version = "~0.2.0-alpha.1", allow-prereleases = true}
 
 #geoapps-utils = {version = "~0.4.0-alpha.1", source = "pypi", allow-prereleases = true}
 geoapps-utils = {git = "https://github.com/MiraGeoscience/geoapps-utils.git", rev = "release/0.4.0"}
+#geoapps-utils = {version = "~0.4.0-alpha.1", allow-prereleases = true}
 
 #mira-simpeg = {version = ">=0.21.2.1-alpha.1, <0.21.2.2.dev", source = "pypi", allow-prereleases = true}
 mira-simpeg = {git = "https://github.com/MiraGeoscience/simpeg.git", rev = "develop"}
+#mira-simpeg = {version = ">=0.21.2.1-alpha.1, <0.21.2.2.dev", allow-prereleases = true}
 
-#param-sweeps = {version = "~0.1.8-alpha.1", source = "pypi", allow-prereleases = true}
+#param-sweeps = {version = "~0.2.0-alpha.1", source = "pypi", allow-prereleases = true}
 param-sweeps = {git = "https://github.com/MiraGeoscience/param-sweeps.git", rev = "develop"}
+#param-sweeps = {version = "~0.2.0-alpha.1", allow-prereleases = true}
 
 ## about pip dependencies
 # to be specified to work with conda-lock
 # - from PyPI: my_package = { version = "1.2.3", source = "pypi" }
+# - from Artifactory : my_package = { version = "1.2.3" }
 # - from URL: !!! no actual lock, as tag or branch can move and installation will fetch current !!!
 #   - for a tag:    my_package = { url = "https://github.com/ORGANISATION/REPO/archive/refs/tags/TAG.tar.gz" }
 #   - for a branch: my_package = { url = "https://github.com/ORGANISATION/REPO/archive/refs/heads/BRANCH.tar.gz" }
 # - to actually lock on a revision: my_package = { git = "https://github.com/ORGANISATION/REPO.git", rev = "GIT_REV" }
 #   (where rev value is a tag, a branch name or a commit hash). Can also use ``branch`` or ``tag`` instead of ``rev``
 
+# indirect package to avoid conflict
+pymatsolver = { version = ">=0.2.0, <0.3.0", source = "pypi" }
+
```

Comment:
**suggestion:** state it the same as in other repos, and move it under `[tool.conda-lock.dependencies]`

```toml
# force some versions to resolve incompatible resolution between PyPI and Conda
#-------------------------------------------------------------------------------
pymatsolver = { version = ">=0.2.0, <0.3.0", source = "pypi" }
```

For example: https://github.com/MiraGeoscience/geopy-distrib/blob/develop/pyproject.toml#L97